### PR TITLE
docker/ci: build RocksDB and Snappy in wercker

### PR DESCRIPTION
**Pull request by vickiniu**
To invoke RocksDB and Snappy in Chain Core, we need to build them in the Docker image that Wercker starts up with. This change installs the dependencies that RocksDB and Snappy need in Docker, and also pulls pre-built static libraries for RocksDB and Snappy into the Docker container.

librocksdb and libsnappy are prebuilt here because they take really long to build, and because we've patched the Snappy project so we can't pull from the git project and build like we do for [protobuf](https://github.com/chain/chain/blob/main/docker/ci/install/install-protobuf#L16).

### CODE REVIEW

File: docker/ci/install/install-rocksdb [hunk 1]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -eo pipefail
+
+# RocksDB dependencies
+# Install gflags
```
Comment (tessr) [inline docker/ci/install/install-rocksdb]: Are you sure we need gflags and zstandard? Especially if we're going to just install snappy and rocksdb from pre-built binaries.

Also, I'm pretty sure I've been building this locally without them.

Comment (vickiniu) [inline docker/ci/install/install-rocksdb]: I'll try running it without them, but it lists gflags and zstandard as dependencies for CentOS (https://github.com/facebook/rocksdb/blob/master/INSTALL.md)

Comment (tessr) [inline docker/ci/install/install-rocksdb]: It's possible you need them to build rocksdb, but not to run it.

Comment (tessr) [inline docker/ci/install/install-rocksdb]: Of course, it's also possible that they're necessary! But let's see.

File: docker/ci/Dockerfile [hunk 2]
```diff
@@ -19,8 +19,9 @@ ENV CHAIN="/go/src/chain" \
 ADD install /usr/bin
 
 RUN yum -y update \
-    && yum install -y autoconf automake bash curl gcc gcc-c++ git libtool make openssl-devel ruby wget unzip \
+    && yum install -y autoconf automake bash bzip2 bzip2-devel curl gcc gcc-c++ git libtool make openssl-devel snappy snappy-devel ruby tar wget unzip \
```
Comment (tessr) [inline docker/ci/Dockerfile]: Ugh, I'm sorry I didn't catch this earlier, but I bet we don't need bzip or snappy here, either.

Comment (vickiniu) [inline docker/ci/Dockerfile]: Without bzip, it ran into a runtime error — but snappy might not be necessary.

Comment (tessr) [inline docker/ci/Dockerfile]: Cool. I bet snappy isn't, since we have the static lib.

**Comment by tessr** [comment]
LGTM!!!!

**Pull request merged**